### PR TITLE
Bundle toastify and transpile it

### PR DESCRIPTION
**Pull request by skjnldsv**
Otherwise it break our apps.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,8 +26,10 @@
   },
   "dependencies": {
     "@nextcloud/l10n": "^1.3.0",
-    "@nextcloud/typings": "latest",
-    "core-js": "^3.6.4"
+    "@nextcloud/typings": "^0.2.2",
+    "babel-loader-exclude-node-modules-except": "^1.0.3",
```
Comment (juliushaertl) [inline package.json:30]: Should be a dev dependency I'd say

**Review by juliushaertl — APPROVED** [review]
Otherwise :+1:

File: package.json (hunk 1 above)
Comment (skjnldsv) [reply package.json:30]: oupsy!

**Comment by korelstar** [comment]
Do I understand correctly that toastify is now included in the npm artifact and apps don't need to install toastify on their own?

If yes: should toastify (and typings and l10n) be a dev dependency? Only then, apps won't have to install those packages ...

**Comment by skjnldsv** [comment]
> If yes: should toastify (and typings and l10n) be a dev dependency? Only then, apps won't have to install those packages ...

Hum, good call!

**Pull request merged**